doc/berrymill.8.md

```diff
@@ -39,20 +38,65 @@ add wanted content.
 
 Important xml tags for derived images are:
 
-* "add": Add specific data to the derived image like a package
+* "add": Add specific data to the derived image like a package, or
+  other elements. Example:
+
+```
+<add>
+   <packages type="iso">
+         <package name="dracut-kiwi-live"/>
+   </packages>
+</add>
+```
+
 * "remove": Remove parts of the original image in the derived image, could be
-  e.g., packages, aggregates...
-* "merge" and "replace": Merge and replace only work on aggregates.
+  e.g., packages, aggregates... Example:
+
+```
+<remove>
+   <packages type="image">
+         <package name="iptables"/>
+   </packages>
+</remove>
+```
+
+* "merge" and "replace": Merge and replace only work on aggregates. One could
+  for example, merge an additional `type` element to the `preferences`:
+
+```
+<merge>
+    <preferences>
+        <type image="typename" primary="true"/>
+    </preferences>
+</merge>
+```
+
 * "remove\_any": Remove any element from description that matches by attributes
-  at least.
-* "set": Set attributes on an element in the derived image.
+   at least. In contrast to remove it does not need precise tag description and
+   is considering attributes to narrow down what tags to remove from the
+   description. Example to remove all users with plain password format:
```

Comment:
Maybe like so:

"remove_any": In contrast to `remove`, this command does not need a precise tag description. It considers attributes to narrow down which tags to remove from the description. For example, it is possible to use it to remove all users configured with plain password format: